airflow/ui/src/components/FlexibleForm/FieldMultiSelect.tsx

```diff
@@ -29,7 +30,9 @@ const labelLookup = (key: string, valuesDisplay: Record<string, string> | undefi
   return key;
 };
 
-export const FieldMultiSelect = ({ name, param }: FlexibleFormElementProps) => {
+export const FieldMultiSelect = ({ name }: FlexibleFormElementProps) => {
+  const { paramsDict, setParamsDict } = useParamStore();
+  const param = paramsDict[name] ?? paramPlaceholder;
```

Comment:
Here (Line 42) the same like for string: If no value is selected, the field value should be `null` as indicator that nothing was selected. At least this is how it was in 2.10.

Reply:
Yeah right now removing all the options will return an empty array which the backend thinks is a valid value even if the field is required.